index.js

```diff
@@ -1,21 +1,21 @@
-// Inspired by (but complete rewrite of) requirejs-metagen
-let _ = require('lodash/fp')
+// Inspired by (but complete rewrite of) requirejs-metagen.
+let _       = require('lodash/fp')
 let Promise = require('bluebird')
-let readDir = Promise.promisify(require('recursive-readdir'))
-let fs = Promise.promisifyAll(require('fs'))
+let readDir = Promise.promisify(require('recursive-readdir')) // Lists all files in a certain directory and all its sub-directories.
```

Comment:
it's because of `fs.readdir`, the standard node method for reading the contents of a directory (which is not recursive)